corehq/messaging/tasks.py

```diff
@@ -89,15 +89,22 @@ def initiate_messaging_rule_run(domain, rule_id):
     transaction.on_commit(lambda: run_messaging_rule.delay(domain, rule_id))
 
 
+def paginated_case_ids(domain, case_type):
+    case_generator = paginate_query_across_partitioned_databases(
+        CommCareCaseSQL,
+        Q(domain=domain, type=case_type, deleted=False),
+        values=['case_id']
+    )
+    for case in case_generator:
+        # return value from case generator is ('pk', 'case_id')
+        yield case[1]
```

Comment:
What do you think of renaming `case` to `row` to make it clear these are not full case objects? Also, this can be changed to `row.case_id` once my other (`named=True`) comment is addressed.